docs/demos/filters/min_max_filter.jl

```diff
@@ -0,0 +1,56 @@
+# Topic : Max Min Filters
+# reference paper : https://core.ac.uk/download/pdf/194053536.pdf
```

Comment:
DemoCards make a relatively strong assumption that the first non-title paragraph is extracted as the description.

Just moving the "In this tutorial we see how can..." part ahead of these should work.

![image](https://user-images.githubusercontent.com/8684355/95279993-88481f80-0886-11eb-842f-83f61f9743b0.png)